packages/nulp_elite/src/configs/urlConfig.json

```diff
@@ -116,6 +116,7 @@
       "CREATE_V2": "user/v2/create",
```

Comment:
@suraj-tekdi - Please make sure follow the stander format of MR title like

Format - 
`Task{{id}} feat/fix : MR title`

Example: 
`Task #0000 feat: Edit user profile page added 2 new fields
`